pkg/proxyagent/agent/manifests/charts/addon-agent/templates/namespace.yaml

```diff
@@ -4,5 +4,5 @@ kind: Namespace
 metadata:
   name: {{ .Release.Namespace }}
   annotations:
-    helm.sh/resource-policy: keep
```

Comment:
Note：this line doesn't actually work since the chart is deployed by addon-framework.